### PR TITLE
GitHub Team Approval Workflow

### PR DESCRIPTION
**Pull request by balajiraman**
Creates a team workflow approval.

### CODE REVIEW

File: .github/workflows/approval-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  labelWhenApproved:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Label approved pull request
+      uses: abinoda/label-when-approved-action@1.0.5
+      env:
+        APPROVALS: "1"
+        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+        ADD_LABEL: "approved"
```
Comment (bvworks) [inline .github/workflows/approval-workflow.yml:15]: Can you change the label to "Code Review Approved" ?

**Review by bvworks — CHANGES_REQUESTED** [review]

File: .github/workflows/approval-workflow.yml (hunk 1 above)
Comment (balajiraman) [reply .github/workflows/approval-workflow.yml:15]: Part of Tutorial, can we do this after this tutorial update?

Comment (balajiraman) [reply .github/workflows/approval-workflow.yml:15]: Can I move forward?

**Review by balajiraman — COMMENTED** [review]
Can I move forward?

**Review by bvworks — CHANGES_REQUESTED** [review]
You did not update the correction I had requested.

**Review by bvworks — APPROVED** [review]
Approved. To be added to backlog for a fix after tutorial.

**Pull request merged**